src/sentry/static/sentry/app/views/settings/projectAlerts/ruleEditor/index.jsx

```diff
@@ -236,7 +236,8 @@ const RuleEditor = createReactClass({
             <AlertRuleRow>
               {t(
                 'Every time %s of these conditions are met:',
-                <Select2Field
+                <SelectField
+                  clearable={false}
```

Comment:
Note that backspaceRemoves and deleteRemoves still default to true whereever you turn off clearable

Reply:
Hmm should we make `backspaceRemoves/deleteRemoves` default to value of `clearable` in `<SelectControl>`? Then allow for it to be explicitly overridden.

Reply:
Sounds good re: backspaceRemoves/deleteRemoves, lets leave the react select default for clearable